core/src/test/scala/org/apache/spark/network/netty/NettyBlockTransferSecuritySuite.scala

```diff
@@ -38,11 +38,20 @@ import org.apache.spark.internal.config.Network
 import org.apache.spark.network.{BlockDataManager, BlockTransferService}
 import org.apache.spark.network.buffer.{ManagedBuffer, NioManagedBuffer}
 import org.apache.spark.network.shuffle.BlockFetchingListener
+import org.apache.spark.network.ssl.SslSampleConfigs
 import org.apache.spark.serializer.{JavaSerializer, SerializerManager}
 import org.apache.spark.storage.{BlockId, ShuffleBlockId}
 import org.apache.spark.util.ThreadUtils
 
 class NettyBlockTransferSecuritySuite extends SparkFunSuite with MockitoSugar with Matchers {
+
+  private def sparkConfWithSsl(): SparkConf = {
+    val conf = new SparkConf()
+    val updatedConfigs = SslSampleConfigs.createDefaultConfigMap()
+    updatedConfigs.entrySet().forEach(entry => conf.set(entry.getKey, entry.getValue))
```

Comment:
I wanted to do that, but `SparkConf` is in `core` and `SslSampleConfigs` is in `network-common` - this would create a dependency loop since we don't have access to `SparkConf` inside `network-common`. If there is another good place to put this helper I can do it.

Reply:
Good point.
Why not `TestUtils` ?

Reply:
@mridulm I took a look but unfortunately I can't -- `TestUtils` is in the main codebase, not the test codebase. `SslSampleConfigs` is test only (See https://github.com/apache/spark/blob/48e207f4a2192d474f2c0f141b984ef0c36a78c3/core/src/main/scala/org/apache/spark/TestUtils.scala#L60-L66)

But I created an `SslTestUtils` and used it everywhere except yarn.

Reply:
Sounds good